crates/task-impls/src/da.rs

```diff
@@ -146,10 +142,9 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, V: Versions> DaTaskState<TYP
             }
             HotShotEvent::DaProposalValidated(proposal, sender) => {
                 let curr_view = self.consensus.read().await.cur_view();
-                if curr_view > proposal.data.view_number() + 1 {
-                    tracing::debug!("Validated DA proposal for prior view but it's too old now Current view {:?}, DA Proposal view {:?}", curr_view, proposal.data.view_number());
-                    return None;
-                }
+                ensure!(curr_view <= proposal.data.view_number() + 1,
+                    format!("Validated DA proposal for prior view but it's too old now Current view {:?}, DA Proposal view {:?}", curr_view, proposal.data.view_number()));
+
```

Comment:
This is now promoted to info level and will show up in data dog.  I think we don't want that

Reply:
now at debug!